### ISSUE TITLE
Joint destroy error when using remove_child.

### ISSUE THREAD

**Issue by Gyrth** [godotengine/godot#25689]
**Godot version:**
3.0.6

**OS/device including version:**
Windows 10

**Issue description:**
Error message about joint destroying when only removing_child.

```
0:00:00:0741 - A body with a joints is destroyed. Please check the implementation in order to destroy the joint before the body.
----------
Type:Warning
Description: 
Time: 0:00:00:0741
C Error: A body with a joints is destroyed. Please check the implementation in order to destroy the joint before the body.
C Source: modules/bullet/rigid_body_bullet.cpp:411
C Function: assert_no_constraints
```
**Steps to reproduce:**

- Create scene with a HingJoint in it.
- Create level scene which has that HingeJoint scene instanced.
- remove_child the HingeJoint scene on_ready.

**Minimal reproduction project:**
[joint_issue.zip](https://github.com/godotengine/godot/files/2842661/joint_issue.zip)

**Comment by LinuxUserGD** [comment]
What happens if you remove_child the joint first?

**Comment by LinuxUserGD** [comment]
With a "joints"?

**Comment by Gyrth** [comment]
I don't want to destroy anything, just remove the child from the scenetree.

**Comment by PLyczkowski** [comment]
Yup, same here. Reparenting stuff with joints breaks them and throws this error...

**Comment by pouleyKetchoupp** [comment]
Fixed by #44703, closing.
